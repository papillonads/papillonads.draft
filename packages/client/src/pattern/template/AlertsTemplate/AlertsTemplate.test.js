import React from 'react'
import alertsTemplateDefault from './AlertsTemplate.render'

jest.mock('react-router-dom', () => {
  return {
    Link: () => <div />,
  }
})

describe('<AlertsTemplate />', () => {
  describe('Snaphot', () => {
    test('must match alertsTemplateDefault', () => {
      expect(global.renderToJSON(alertsTemplateDefault)).toMatchSnapshot()
    })
  })
})
