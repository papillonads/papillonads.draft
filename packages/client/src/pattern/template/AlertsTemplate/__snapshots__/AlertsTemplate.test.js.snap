// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AlertsTemplate /> Snaphot must match alertsTemplateDefault 1`] = `
<div>
  <h2 />
  Under construction
   
  <span
    aria-label="emoji"
    role="img"
  >
    ðŸ™‚
  </span>
  <div />
</div>
`;
